apps/prairielearn/src/lib/editors.ts

```diff
@@ -973,6 +991,46 @@ export class CourseInstanceCopyEditor extends Editor {
   }
 }
 
+async function updateInfoAssessmentFilesForInstanceCopy(
+  courseInstancePath: string,
+  fromCourseSharingName: string,
+) {
```

Comment:
Should this function update all assessments to use new UUIDs? I guess we don't strictly need to since they're only unique within a course instance, and we're already making a brand new course instance here.

Reply:
Right, that's why I figured it was fine.